server/api-util/lineItems.js

```diff
@@ -0,0 +1,53 @@
+const {
+  calculateQuantityFromDates,
+  calculateTotalFromLineItems,
+  consrtuctValidLineItems,
```

Comment:
Typo "construct"

Reply:
It seems that this function is not used in this file (Still it's best to fix the typo elsewhere)